xbmc/cores/dvdplayer/DVDPlayerRadioRDS.cpp

```diff
@@ -418,23 +418,23 @@ const std::string piRDSLanguageCodes[128]=
 /* ----------------------------------------------------------------------------------------------------------- */
 
 #define EntityChars 56
-const char *entitystr[EntityChars]  = { "&apos;",   "&amp;",    "&quot;",  "&gt",      "&lt",      "&copy;",   "&times;", "&nbsp;",
-                                        "&Auml;",   "&auml;",   "&Ouml;",  "&ouml;",   "&Uuml;",   "&uuml;",   "&szlig;", "&deg;",
-                                        "&Agrave;", "&Aacute;", "&Acirc;", "&Atilde;", "&agrave;", "&aacute;", "&acirc;", "&atilde;",
-                                        "&Egrave;", "&Eacute;", "&Ecirc;", "&Euml;",   "&egrave;", "&eacute;", "&ecirc;", "&euml;",
-                                        "&Igrave;", "&Iacute;", "&Icirc;", "&Iuml;",   "&igrave;", "&iacute;", "&icirc;", "&iuml;",
-                                        "&Ograve;", "&Oacute;", "&Ocirc;", "&Otilde;", "&ograve;", "&oacute;", "&ocirc;", "&otilde;",
-                                        "&Ugrave;", "&Uacute;", "&Ucirc;", "&Ntilde;", "&ugrave;", "&uacute;", "&ucirc;", "&ntilde;" };
-const char *entitychar[EntityChars] = { "'",        "&",        "\"",      ">",        "<",         "c",        "*",      " ",
-                                        "Ä",        "ä",        "Ö",       "ö",        "Ü",         "ü",        "ß",      "°",
-                                        "À",        "Á",        "Â",       "Ã",        "à",         "á",        "â",      "ã",
-                                        "È",        "É",        "Ê",       "Ë",        "è",         "é",        "ê",      "ë",
-                                        "Ì",        "Í",        "Î",       "Ï",        "ì",         "í",        "î",      "ï",
-                                        "Ò",        "Ó",        "Ô",       "Õ",        "ò",         "ó",        "ô",      "õ",
-                                        "Ù",        "Ú",        "Û",       "Ñ",        "ù",         "ú",        "û",      "ñ" };
+static const char *entitystr[EntityChars]  = { "&apos;",   "&amp;",    "&quot;",  "&gt",      "&lt",      "&copy;",   "&times;", "&nbsp;",
```

Comment:
Think the best is to leave independent, the positions of "entitychar" depends on "entitystr" and this to given text, to bring them together on HTMLUtil make the code style bigger as the table itself here.